pkg/tasks/probes.go

```diff
@@ -790,12 +795,11 @@ func detectCCMMigrationStatus(s *state.State) (*state.CCMStatus, error) {
 				if csiFlagRegex.MatchString(command) {
 					status.CSIMigrationEnabled = true
 				}
```

Comment:
Set unconditionally to `true` for providers that previously had the in-tree cloud provider.